locale/en_US/translations.ts

```diff
@@ -692,5 +692,25 @@
             <source>Play Trailer</source>
             <translation>Play Trailer</translation>
         </message>
+        <message>
+            <source>Attempt H.264 Direct Play (Profile Lvl)</source>
+            <translation>Attempt H.264 Direct Play (Profile Lvl)</translation>
+            <extracomment>Settings Menu - Title for option</extracomment>
+        </message>
+        <message>
+            <source>Attempt Direct Play for H.264 media with unsupported profile levels (> 4.2) before falling back to transcoding if it fails.</source>
+            <translation>Attempt Direct Play for H.264 media with unsupported profile levels (> 4.2) before falling back to transcoding if it fails.</translation>
+            <extracomment>Settings Menu - Description for option</extracomment>
+        </message>
+        <message>
+            <source>Attempt HECV Direct Play (Profile Lvl)</source>
```

Comment:
```suggestion
            <source>Attempt HEVC Direct Play (Profile Lvl)</source>
```

Also line below.

What does "Profile Lvl" mean?